images/memes/debug.log

```diff
@@ -0,0 +1,10 @@
+[1130/142031.212:INFO:CONSOLE(1)] "Long rAF frame-time", source: file:///C:/Program%20Files/Adobe/Adobe%20Photoshop%202020/Required/www-rich-tooltips/build/external.js (1)
+[1201/075913.015:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1201/075918.030:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1201/075923.045:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1201/075928.056:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1201/075933.067:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1201/075938.086:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1201/080010.816:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1201/080015.836:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
+[1202/153916.398:WARNING:dns_config_service_win.cc(668)] Failed to read DnsConfig.
```

Comment:
wut da hek

After this is removed this is good to go

Reply:
think this is done now